docs/static/ds4/skin-full.css

```diff
@@ -375,28 +427,66 @@ svg.expand-btn__icon:only-child {
   display: flex;
   margin: 0;
 }
+.btn__cell--fixed-height span.btn__icon,
+.expan-btn__cell--fixed-height span.expan-btn__icon,
```

Comment:
Small typo here. Amazed I spotted this!

Reply:
You actually didn't catch _both_ of them. ;-)